packages/contracts/contracts/CyberDropBase.sol

```diff
@@ -124,27 +109,18 @@ contract CyberDropBase is CyberTokenBase {
 
     if (drop.amountCap != 0) {
       require(drop.minted.current() < drop.amountCap, 'CR');
+      require(drop.amountCap - drop.minted.current() >= _quantity, 'CR');
     }
 
     require(
       block.timestamp > drop.timeStart && block.timestamp <= drop.timeEnd,
       'OOT'
     );
-    uint256 timeSpent = block.timestamp - drop.timeStart;
-    uint256 duration = drop.timeEnd - drop.timeStart;
-    uint256 price = getPriceFor(
-      timeSpent,
-      duration,
-      drop.priceStart,
-      drop.priceEnd,
-      drop.stepDuration
-    );
-    require(msg.value >= price, 'IA');
-    uint256 amountOnCyber = (msg.value * drop.shareCyber) / 100;
-    uint256 amountCreator = msg.value - amountOnCyber;
+
+    require(msg.value >= drop.price, 'IA');
```

Comment:
* quantity? And now can be a strict equality

Reply:
Nice

Reply:
@getspooky